types/src/database_sql.rs

```diff
@@ -1631,6 +1623,45 @@ impl Database {
                 .await?,
         )
     }
+
+    pub async fn get_menu_by_slug(&self, slug: String) -> Result<Menu> {
+        let menu = query_file!("queries/menu_by_slug.sql", slug)
+            .fetch_one(&self.client)
+            .await?;
+
+        let m_id: Uuid = menu.id;
```

Comment:
all three of these `m_` fields seem needlessly redundant since you're only using these values in 1639-41. Is there a reason you did this rather than just 
```
Ok(Menu {
        id: menu.id,
        name: menu.name,
        slug: menu.slug,
        items,
})